pkg/storage/kube/utils.go

```diff
@@ -0,0 +1,20 @@
+package kube
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetStorageClassNames returns the names of the StorageClass instances in the cluster
+func (s *store) GetStorageClassNames() ([]string, error) {
+	scl, err := s.client.StorageV1().StorageClasses().List(meta.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	var scss []string
```

Comment:
nit: I don't particularly like the naming, but don't mind me 😆

Reply:
It was ‘GetStorageClasses’ in the beginning, had to rename that when I thought that we only need their names... Feel free to suggest alternative naming :)

Reply:
Nah, it's fine, really - it's a local variable, not exposed anywhere, so it doesn't impact anything.